guides/common/modules/proc_extending-the-discovery-image.adoc

```diff
@@ -32,16 +32,20 @@ You can also provide a compressed archive file containing extra code for the ima
 ----
 # zip -r my_extension.zip .
 ----
-. To inform the Discovery image of the extensions it must use, place your zip files on your TFTP server with the Discovery image, and then update the `APPEND` line of the PXELinux template with the `fdi.zips` option where the paths are relative to the TFTP root.
+. Inform the Discovery kernel of the extensions it must use.
+Place your zip files on your TFTP server with the Discovery image and customize the Discovery PXE boot with the `fdi.zips` parameter where the paths are relative to the TFTP root.
++
 For example, if you have two archives at `$TFTP/zip1.zip` and `$TFTP/boot/zip2.zip`, use the following syntax:
 +
 [options="nowrap" subs="+quotes,verbatim"]
 ----
 fdi.zips=zip1.zip,boot/zip2.zip
 ----
++
+For more information, see xref:Customizing_the_Discovery_PXE_Boot_{context}[].
 
 You can append new directives and options to the existing environment variables (`PATH`, `LD_LIBRARY_PATH`, `RUBYLIB` and `FACTERLIB`).
 If you want to specify the path explicitly in your scripts, the `.zip` file contents are extracted to the `/opt/extension` directory on the image.
 
 You can create multiple `.zip` files but be aware that they are extracted to the same location on the Discovery image.
-Files extracted from in later `.zip` files overwrite earlier versions if they have the same file name.
+Files extracted from in later `.zip` files overwrite earlier versions if they contain the same file name.
```

Comment:
```suggestion
Files extracted from later version `.zip` files overwrite earlier versions if they contain the same file name.
```
Is there a better way to say `in later .zip files`?

Reply:
It's not about a later version, it's about having multiple .zip files.